.github/ISSUE_TEMPLATE/Security_vulnerability.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Security vulnerability
+about: Do not report security vulnerabilities using GitHub issues. Please send an email to `callback@ibm.com` instead.
+labels: security
+
+---
+
+<!-- 🚨 STOP 🚨 STOP 🚨 STOP 🚨
+
+BE RESPONSIBLE WHEN REPORTING SECURITY VULNERABILITIES.
+USE A PRIVATE CHANNEL TO ALLOW THE VULNERABILITY TO BE
+FIXED BEFORE IT'S PUBLICIZED.
+
+Please send an email to `callback@ibm.com` with:
```

Comment:
I think the email to use is `reachsl@us.ibm.com`.

Reply:
Could you please double-check? I took the email value from our loopback-next template, see here:

https://github.com/strongloop/loopback-next/blob/master/.github/ISSUE_TEMPLATE/Security_vulnerability.md

Reply:
yes. I can confirm that `reachsl@us.ibm.com` is the right one to use.